label_sync/config_example.sh

```diff
@@ -1,5 +1,5 @@
 # Which file to read the labels.yaml from
-export LABEL_SYNC_CONFIG=labels.yaml
+export LABEL_SYNC_CONFIG=../prow/config/labels.yaml
```

Comment:
What's going on here? Is this related?

Reply:
Yes, labels.yaml was moved to config/prow to match how openshift/release has things setup. prow reads that file.

I only want reviewers or org members to be able to use `/label ci`, I'm hoping the labels.yaml is what prow uses for that. I can't find docs though.